pkg/release/candidate.go

```diff
@@ -0,0 +1,202 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package release
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	kgit "k8s.io/release/pkg/git"
+	"k8s.io/release/pkg/util"
+)
+
+// Candidate is the main structure for pushing builds.
+type ffOptions struct {
+	branch         string
+	mainRef        string
+	nonInteractive bool
+}
+
+type rootOptions struct {
+	nomock   bool
+	cleanup  bool
+	repoPath string
+	logLevel string
+}
```

Comment:
They would have to be public I guess. I think we could also combine them into one structure.

Reply:
Ok, will change these to public and combine both into a one struct, what will be name , just `Options` ?

Reply:
@saschagrunert Combined to a new struct, please take a look.